### DISCUSSION TITLE
Values from checkboxes stored as array with binding to storage

### DISCUSSION THREAD

**Discussion by pythonormen** [zauberzeug/nicegui#1117] [Ideas / Feature Requests]
### Question

Hi,

Is this supported with Nicegui?
https://quasar.dev/vue-components/checkbox#array-model

I.e. I'd like to do the same as below but with checkboxes.

Thank you

```python
ui.select(['One', 'Two', 'Three'], multiple=True) \
    .bind_value(app.storage.general, 'numbers') \
    .props('dense use-chips clearable filled')
```

**Comment by falkoschindler** [comment]
Hi @pythonormen!
To answer your question: No, it's currently not supported.
I'm also thinking about multiple checkboxes every now and then. But I always struggle with how the API might look like (and how to implement it). We might need a new element `ui.checkboxes` which is bindable against a list.
I'll convert the discussion into a feature request to further think about a possible implementation.
